exdb/forms.py

```diff
@@ -99,25 +103,24 @@ def clean(self):
             (needs_verification and not self.approval_form and
                 not self.cleaned_data.get('next_approver'),
              ValidationError(_('Please select the supervisor to review this experience'))),
+            (needs_verification is False and (self.cleaned_data.get('start_datetime', max_dt) > self.when),
+                ValidationError(_('This experience must have a date in the past'))),
+            (needs_verification and (self.cleaned_data.get('start_datetime', max_dt) < self.when),
+                ValidationError(_('This experience must have a date in the Future'))),
```

Comment:
Also, both of these error messages should probably say "start date" instead of just "date".